src/inc/corinfo.h

```diff
@@ -213,11 +213,11 @@ TODO: Talk about initializing strutures before use
     #define SELECTANY extern __declspec(selectany)
 #endif
 
-SELECTANY const GUID JITEEVersionIdentifier = { /* e5708e9e-dd18-4287-8745-5d10ff2697cf */
-    0xe5708e9e,
-    0xdd18,
-    0x4287,
-    { 0x87, 0x45, 0x5d, 0x10, 0xff, 0x26, 0x97, 0xcf }
+SELECTANY const GUID JITEEVersionIdentifier = { /* 1348cf5b-d47f-4102-87a7-68020d32d677 */
```

Comment:
I pushed the new commit (without the GUID update) to the source branch of this pull request. However, the pull request is not updated yet (maybe, some failure in github operation).
The branch: https://github.com/ruben-ayrapetyan/coreclr/commits/pass-template-mt-to-new-arr-helper